.github/workflows/ci.yml

```diff
@@ -83,6 +83,8 @@ jobs:
       - uses: ./
         with:
           tool: ${{ steps.tool-list.outputs.tool }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The fact that cargo-binstall's performance or robustness would be worse without this env var at a high probability is very unfortunate.

Reply:
Yeah, that's something I would like to improve on.

Maybe we should serailise the github requests instead, that would be faster.